packages/block-editor/src/components/rich-text/index.js

```diff
@@ -214,13 +214,24 @@ export function RichTextWrapper(
 
 			const { getBlockAttributes } = select( blockEditorStore );
 			const blockAttributes = getBlockAttributes( clientId );
-			const fieldsList = blockBindingsSource?.getFieldsList?.( {
-				select,
-				context: blockBindingsContext,
-			} );
+			let clientSideFieldLabel = null;
+			if ( blockBindingsSource?.editorUI ) {
+				if ( blockBindingsSource.usesContext?.length ) {
+					for ( const key of blockBindingsSource.usesContext ) {
+						blockBindingsContext[ key ] = blockContext[ key ];
+					}
+				}
```

Comment:
I see you've addressed this in https://github.com/WordPress/gutenberg/pull/71542/commits/5c290980b59d178533dbb5b020add6cb96c37a37 (by changing `editorUIContext` to `blockBindingsContext`). Thanks for that!

But I thought we can maybe drop these lines altogether? I believe `blockBindingsContext` is already prepopulated with the relevant information [further up](https://github.com/WordPress/gutenberg/pull/71542/files#diff-bec072c72a2f961596701aef67f67575b0a3b918135a6587229a0cd79bb3db44R192-R196).

 ```suggestion
```